honeybee_energy/writer.py

```diff
@@ -22,44 +28,368 @@ def generate_idf_string(object_type, values, comments=None):
         ep_str = object_type + ',\n ' + '\n '.join(
             '{},{}!- {}'.format(val, space, com) for val, space, com in
             zip(values[:-1], spaces[:-1], comments[:-1]))
-        ep_str = ep_str + '\n {};{}!- {}'.format(
-            values[-1], spaces[-1], comments[-1])
+        if len(values) == 1:
+            ep_str = ep_str + '{};{}!- {}'.format(values[-1], spaces[-1], comments[-1])
```

Comment:
This is not that much of text concatenate but if you look online people have strong opinion about concatenating strings in Python.

Reply:
What alternative are you suggesting?

Reply:
I see people prefer different ways but I still don't see a better solution for this example.